vision/text/detect_words.py

```diff
@@ -119,7 +130,7 @@ def _get_rotated_min_area_rect(
         )
```

Comment:
A couple of things unrelated to this PR but I want to get the code cleaned up since you are already working on it.
Can't add my comment on the exact line number but on line 118 there is a magic number `depth_image < 8000` that should be a constant.
Also the parameters for cv2 functions in this block should be keyword-specified for clarity on Canny, GuassianBlur, and findContours.